internal/utils.go

```diff
@@ -217,6 +217,12 @@ func getPolicyTemplates(policyConf *types.PolicyConfig) ([]map[string]interface{
 					annotations[severityAnnotation] = policyConf.Severity
 
 					policyTemplateUnstructured.SetAnnotations(annotations)
+
+					err := setGatekeeperEnforcementAction(policyTemplate,
```

Comment:
It might be cleaner to directly work with the manifest rather than after it is a policy template. You could do this right at the start of the `for _, manifest := range manifestGroup {` loop.